projects/storefrontapp-e2e-cypress/cypress/integration/vendor/cds/merchandising-carousel.e2e-spec.ts

```diff
@@ -97,377 +104,392 @@ describe('Merchandising Carousel', () => {
     });
   });
 
-  it('should send another request to the strategy service, if consent is accepted, containing the consent-reference as a HTTP header', () => {
-    testHomePage();
+  describe('with consent granted', () => {
+    beforeEach(() => {
+      cy.route('POST', '/edge/clickstreamEvents').as(carouselEventRequestAlias);
+
+      cy.visit(
+        `/${merchandisingCarousel.DEFAULT_LANGUAGE}/${merchandisingCarousel.DEFAULT_CURRENCY}`
+      );
+      merchandisingCarousel.verifyRequestToStrategyService(
+        strategyRequestAlias,
+        {}
+      );
+      profileTagHelper.grantConsent();
```

Comment:
this fails at times as the page wasn't done loading making it wait for x amount of time

this relates to my comment about the .visit and should be changed in necessary places that uses `cy.visit` and utilize `waitForPage` function